Tribler/Core/Modules/restapi/channels/channels_torrents_endpoint.py

```diff
@@ -70,11 +71,15 @@ def render_GET(self, request):
         results_local_torrents_channel = self.channel_db_handler\
             .getTorrentsFromChannelId(channel_info[0], True, torrent_db_columns)
 
+        should_filter = self.session.tribler_config.get_family_filter_enabled()
+        if 'disable_filter' in request.args and len(request.args['disable_filter']) > 0 \
+                and request.args['disable_filter'][0] == "1":
+            should_filter = False
+
         results_json = []
         for torrent_result in results_local_torrents_channel:
             torrent_json = convert_db_torrent_to_json(torrent_result)
-            if (self.session.tribler_config.get_family_filter_enabled() and
-                    self.session.lm.category.xxx_filter.isXXX(torrent_json['name'])) or torrent_json['name'] is None:
+            if (should_filter and torrent_json['category'] == 'xxx') or torrent_json['name'] is None:
```

Comment:
@brussee did you notice the parenthesis? `should_filter` has no influences on the torrent name, correct me if I'm wrong.